google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStub.java

```diff
@@ -1356,10 +1355,17 @@ public Map<String, String> extract(ExecuteQueryRequest executeQueryRequest) {
     ServerStreamingCallable<ExecuteQueryCallContext, SqlRow> withBigtableTracer =
         new BigtableTracerStreamingCallable<>(merging);
 
+    ServerStreamingCallSettings<ExecuteQueryCallContext, SqlRow> retrySettings =
+        ServerStreamingCallSettings.<ExecuteQueryCallContext, SqlRow>newBuilder()
+            // TODO resumption strategy and retry settings
+            .build();
+
+    ServerStreamingCallable<ExecuteQueryCallContext, SqlRow> retries =
```

Comment:
Will you put a comment noting that this is just for client side metrics and we don't want any retries to happen right now?